### PR TITLE
Feature: Driver - duckdb

### PR DESCRIPTION
**Pull request by oscar60310**
Driver for [duckdb](https://duckdb.org)

Document:
https://github.com/Canner/vulcan/tree/feature/driver-duckdb/packages/extension-driver-duckdb

I deleted the extension for lab env because it can use this driver instead.

### CODE REVIEW

**Review by kokokuo — APPROVED** [review]
Beside some parts, others LGTM 👍

File: labs/playground1/Makefile [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
-	@yarn; \
-	cd ./playground-exts; \
-	yarn link; \
-	cd ..; \
-	yarn link "playground-exts"
+	@yarn;
 
 # build the required packages
-build: pkg-core pkg-build pkg-serve pkg-cli
+build: pkg-core pkg-build pkg-serve pkg-cli pkg-serve pkg-extension-driver-duckdb
```
[the remark this answers is not in the document]
Comment (oscar60310) [reply labs/playground1/Makefile:13]: Yes, typo here, thanks!

File: types/duckdb.d.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    prepare(sql: string): Statement;
+    /**
+     * Convenience method for Connection#run using a built-in default connection
+     * @param sql
+     * @param params
+     */
+    run(sql: string, ...params: any): void; // This last parameter should be callback function.
+    /**
+     * Convenience method for Connection#apply using a built-in default connection
```
[the remark this answers is not in the document]
Comment (oscar60310) [reply types/duckdb.d.ts]: Yes, thanks.

**Comment by oscar60310** [comment]
Hi @kokokuo , all issues have been fixed.

**Pull request merged**